src/01-general/invites.apib

```diff
@@ -0,0 +1,34 @@
+## Invites [/invites]
+
+*Required scopes: `users`*
+
+### invites.list [GET /invites.list]
+
+Get a list of all invites that have been sent.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + email: `john@teamleader.eu` (string, optional)
+
++ Response 200 (application/json)
+    + Attributes
+        + data (object)
+            + id: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5` (string)
+            + first_name: `john` (string)
+            + last_name: `doe` (string)
+            + email: `john@teamleader.eu` (string)
+            + created_at: `2017-05-09T11:25:11+00:00` (string)
```

Comment:
Q: do we need the time here or the date is just enough?

Reply:
That would be up to the front-end to decide. How we have it in the API response doesn't really matter, the more detailed the better I'd say.

Reply:
We'll have to call it `invited_at` though.